app/assets/stylesheets/notifications-welcome-broadcast.scss

```diff
@@ -0,0 +1,20 @@
+// TODO: [@thepracticaldev/delightful]: Refactor nesting in this file
+// in addition to notifications.scss so that the nesting is
+// not as deep as it currently is
+.notifications-index {
+  .home {
+    .articles-list {
+      .single-article {
+        .content {
+          &.broadcast-content {
+            .opt-out {
+              padding-top: 24px;
+              font-size: 12px;
```

Comment:
ProTip for your next PR: check out `app/assets/stylesheets/config/_variables.scss`. I think these styles were added _before_ crayons, which is why they aren't using those variables yet. But now, you should be able to replace these with `var(--su-6)` and `var(--fs-xs)` respectively! 😉

Reply:
Ah, thank you for pointing that out @vaidehijoshi! 😄